### PR TITLE
Cleanup E0412 and E0422

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0422.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-You are trying to use an identifier that is either undefined or not a struct.
+An identifier that is either undefined or not a struct was used.
```
Comment (Dylan-DPC) [inline src/librustc_error_codes/error_codes/E0422.md]:
```suggestion
An identifier that is neither undefined nor a struct was used.
```

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

**Comment by pietroalbini** [comment]
Had to kill the PR build to get CI back up and running, sorry for the trouble.

**Comment by Dylan-DPC** [comment]
No worries :)

**Comment by JohnTitor** [comment]
Failed in https://github.com/rust-lang/rust/pull/69979#issuecomment-598860236
You should bless `ui/lint/use_suggestion_json.rs` test, it includes E0412 description.

@bors r-

**Pull request merged**